src/App.js

```diff
@@ -1,29 +1,31 @@
-import React, {useEffect, useContext} from 'react';
-import {BrowserRouter, Switch} from 'react-router-dom';
+import React, { useEffect, useContext, useState } from 'react';
+import { BrowserRouter, Switch } from 'react-router-dom';
```

Comment:
These spaces must be removed by prettier…

Reply:
i have prettier... but it keeps saving with the space for App.js 😔💁

Reply:
mm
Do you have a `.prettierrc` file on your root?
```json
{
  "printWidth": 120,
  "semi": true,
  "singleQuote": true,
  "trailingComma": "all",
  "bracketSpacing": false,
  "jsxBracketSameLine": true,
  "arrowParens": "avoid",
  "tabWidth": 2,
  "useTabs": false
}
```

Reply:
And have you installed ESlint for VSCode?

Reply:
yes, i have the same on my rules for prettier on my .prettierrc

also, just installed ESlint, thanks ^^